addon/ActivityStreams.js

```diff
@@ -142,24 +151,12 @@ ActivityStreams.prototype = {
    * Broadcast a message to all workers
    */
   broadcast(action) {
+    this._store.dispatch(action);
```

Comment:
This change ensures that broadcast will dispatch even if no tabs are currently open.